modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -73,8 +73,18 @@ You can install {product-title} version {product-version} on the following IBM h
 [discrete]
 === Hardware requirements
 
-* 1 LPAR with 6 IFLs that supports SMT2
-* 1 OSA or RoCE network adapter
+* The equivalent of 6 IFLs, which are SMT2 enabled, for each cluster.
+* At least one network connection to connect to the LoadBalancer service and for traffic outside the cluster.
+
+[NOTE]
+====
+You can use dedicated or shared IFLs to assign sufficient compute resources. Resource sharing is one of the key strengths of IBM Z. However, you must adjust capacity correctly on each hypervisor layer and ensure sufficient resources for every OpenShift cluster.
+====
+
+[IMPORTANT]
+====
+Important: The LPARs that are used to setup the {product-title} clusters must provide sufficient compute capacity, since the overall performance of the cluster can be impacted. In this context, topics like LPAR weights, entitlements, and CPU shares on hypervisor level play an important role. 
```

Comment:
I wouldn't use "topics" because there are existing references to "topics" in the documentation sense in the collection, and it's best to use a word in only one way whenever possible.

Also, is there a way to not have back-to-back notes? Can you move the preceding note or turn it into plain text?

```suggestion
Since the overall performance of the cluster can be impacted, the LPARs that are used to setup the {product-title} clusters must provide sufficient compute capacity. In this context, system attributes like LPAR weights, entitlements, and CPU shares on the hypervisor level play an important role. 
```